lib/ExpressionUtils/ExpressionUtilsImpl.py

```diff
@@ -274,8 +280,8 @@ def upload_expression(self, ctx, params):
 
         genome_ref = self._get_genome_ref(assembly_or_genome_ref, params)
 
-        expression_levels, tpm_expression_levels = self._get_expression_levels(source_dir, 
-                                                                               genome_ref)
+        expression_levels, tpm_expression_levels = self._get_expression_levels(
+            source_dir, genome_ref, params.get('transcripts'))
```

Comment:
Add a constant for parameter string (PARAM_IN_TRANSCRIPTS) in the beginning similar to how it is done for other input parameters?